CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- RedHat-certified version of the secrets-provider-for-k8s image [cyberark/secrets-provider-for-k8s#93](https://github.com/cyberark/secrets-provider-for-k8s/pull/93)
```

Comment:
i think it should be `([cyberark/secrets-provider-for-k8s#93](https://github.com/cyberark/secrets-provider-for-k8s/pull/93))`

Reply:
https://github.com/cyberark/conjur-authn-k8s-client/edit/master/CHANGELOG.md looking at other projects I see we do a combo. ([]()) is better?

Reply:
yes, that's our standard

Reply:
so i will update our whole changelog in that fashion. separate PR?

Reply:
only if it's easy, thanks